alot/commands/envelope.py

```diff
@@ -146,8 +146,8 @@ def apply(self, ui):
                 ui.apply_command(globals.FlushCommand())
                 ui.apply_command(commands.globals.BufferCloseCommand())
             except DatabaseError as e:
-                logging.error(e.message)
-                ui.notify('could not index message:\n%s' % e.message,
+                logging.error(str(e))
+                ui.notify('could not index message:\n%s' % str(e),
```

Comment:
If it's implemented the way I think it is then it will. I'll run a quick test to ensure that you're right, but I'm pretty sure you are. I'll update